src/physics/cam/clubb_intr.F90

```diff
@@ -1893,11 +1893,14 @@ real(r8) :: inv_exner_clubb(pcols,pverp)     ! Inverse exner function consistent
      qitend(:ncol,:)=0._r8
      initend(:ncol,:)=0._r8
 
-     call ice_macro_tend(naai(:ncol,top_lev:pver),state1%t(:ncol,top_lev:pver), &
-        state1%pmid(:ncol,top_lev:pver),state1%q(:ncol,top_lev:pver,1),state1%q(:ncol,top_lev:pver,ixcldice),&
-        state1%q(:ncol,top_lev:pver,ixnumice),latsub,hdtime,&
-        stend(:ncol,top_lev:pver),qvtend(:ncol,top_lev:pver),qitend(:ncol,top_lev:pver),&
-        initend(:ncol,top_lev:pver))
+     do k = top_lev, pver
+        do i = 1, ncol
+           call ice_macro_tend(naai(i,k),state1%t(i,k),state1%pmid(i,k),      &
```

Comment:
Do we want to create a vectorized version of ice_macro_tend?

Reply:
Thanks John. Just push the vectorized version of ice_macro_tend.